src/Power.cpp

```diff
@@ -234,6 +234,9 @@ bool Power::setup()
 void Power::shutdown()
 {
 
+#if defined(USE_EINK)
+    digitalWrite(PIN_EINK_EN, LOW); //power off backlight first
+#endif
```

Comment:
Can you please add an additional macro guard to make sure PIN_EINK_EN is defined before you use it? Asking for a hypothetical new device with EINK but without that PIN Definition.